keycloak/keycloak-operator
# #485 Make ImagePullPolicy configurable

## Related GH Issue
Closes #485 

## Additional Information
- InitContainers & Containers of keycloak & postgres can have an ImagePullPolicy as user input in the CRD
- Unit tests updated
- pre-commit hook updated go.mod & go.sum
- Example in deploy/examples/keycloak/keycloak.yaml

## Verification Steps

1. make setup -> make code/gen -> kubectl apply -f deploy/examples/keycloak/keycloak.yaml
2. Check the imagePullPolicy in the keycloak statefulSet

## Checklist:
